wiki/Configuration:-Layout.md

```diff
@@ -221,6 +221,18 @@ layout {
 }
 ```
 
+Gradients can be rendered with different kinds of color interpolation, this doesen't mean that the arguments the gradient takes
```

Comment:
Markdown files put newline after every sentence and not in other places. For lists use proper markdown lists.